python/ray/services.py

```diff
@@ -19,6 +20,11 @@
 import ray.ray_constants as ray_constants
 import psutil
 
+libc = None
+prctl = None
+PR_SET_PDEATHSIG = 1
+SIGKILL = 9
```

Comment:
How about `int(signal.SIGKILL)` instead of defining our own constant

Reply:
Oops, didn't realize it's in `signal`, yeah good idea.

Reply:
Oh actually that means I have to import locally, since `signal` isn't available on Windows. I guess I can do that, not sure if you prefer it though.

Reply:
Oh wait just kidding, only `SIGKILL` isn't available on Windows.

Reply:
Pushed